### DISCUSSION TITLE
Github Search functionality upto 2500 and 100 per page

### DISCUSSION THREAD

**Discussion by djassie** [community/community#17923] [Other Feature Feedback, Questions, & Ideas]
Github is a awesome place to store code repositories.

But, when searching anything - it has limited visibility to fetch only 1000 results.

Can we have 50 - 100 results per page and upto 25000 results.

It’s a professional place.

**Comment by look** [comment]
@djassie can you let us know what use case you have that you need to return 25,000 results?

**Reply by djassie** [reply]
For research purpose. (Finding needle in heystack, cause developers barely tag properly, so anything important comes)

Example: I am developer constantly trying to find useful ones. Now, devs who create repositories, some of them are rookie level, some of them are production level. 

Now, most of the developers barely tags, and I want to find something useful. Because of search results are so low in number, most of the results could not be discuvered. 

So, my only purpose is to explore important respositories and learn them. What other developers are doing in my field.
